batchflow/models/tf/encoder_decoder.py

```diff
@@ -163,10 +170,16 @@ def body(cls, inputs, name='body', **kwargs):
     def head(cls, inputs, targets, name='head', **kwargs):
         """ Linear convolutions. """
         kwargs = cls.fill_params('head', **kwargs)
+
         with tf.variable_scope(name):
             x = cls.crop(inputs, targets, kwargs['data_format'])
+            x = super().head(x, name, **kwargs)
+
```

Comment:
You don't need extra scope named `name` inside already named `name` scope

Reply:
Couldn't find better name, so leaving this as is